simtools/config.py

```diff
@@ -74,7 +74,7 @@ def get(par):
         return config[par]
 
 
-def collectConfigArg(name, value):
+def collectConfigArg(name, value=None):
```

Comment:
I was actually misusing this function.
The more meaningful function is cfg.get that returns an entry from the config file. This is the one that I should be using here (it is fixed now).

There are situations in which a class can receive an argument that is in the config file. If the argument is given, it is used as it is. If it is not given (defaul = None), we collect it from the config file. For example, some classes receive modelFilesLocations, in case you want to use something different from the config file.

Instead of doing
 if modelFilesLocation is None:
   self.modelFilesLocation = cfg.get('modelFilesLocations')
else:
   self.modelFilesLocation = modelFilesLocations,

I wrote this function collectConfigArg. It's very simple, just meaning to clean the code a bit.

Reply:
OK, looks useful, but I would then suggest to expand a bit the doc to explain it and also change the name. This is not really "collecting", it is still "getting", so I would call it "getConfigPar" instead (also par instead of arg cause it's not an input argument).

Reply:
Yes, I'll fix that later.

Reply:
OK!